examples/resources/maas_configuration/import.sh

```diff
@@ -0,0 +1,2 @@
+# Import a MAAS configuration setting using the key as the identifier
+terraform import maas_configuration.test release_notifications
```

Comment:
the usual suspect nit: Please add a new line here and re-create docs. If we end up removing import functionality, feel free to delete the file instead.